muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/actions/workspaces/index.ts

```diff
@@ -267,47 +267,51 @@ let setCurrentWorkspace:SetCurrentWorkspaceTriggerType = function setCurrentWork
       let isCourseMember:boolean;
       let journals:WorkspaceJournalsType;
       let details:WorkspaceDetailsType;
+      let chatStatus: WorkspaceChatStatusType;
       let status = getState().status;
-      [workspace, assesments, feeInfo, assessmentRequests, activity, additionalInfo, contentDescription, producers, isCourseMember, journals, details] = await Promise.all([
-                                                 reuseExistantValue(true, workspace, ()=>promisify(mApi().workspace.workspaces.cacheClear().read(data.workspaceId), 'callback')()),
+      [workspace, assesments, feeInfo, assessmentRequests, activity, additionalInfo, contentDescription, producers, isCourseMember, journals, details, chatStatus] = await Promise.all([
+                                                reuseExistantValue(true, workspace, ()=>promisify(mApi().workspace.workspaces.cacheClear().read(data.workspaceId), 'callback')()),
```

Comment:
Indent of 48 is a bit excessive, eh?

Reply:
Self formatting... I think we shouldn't worry about it now, it just happened when using eclipse.